### ISSUE TITLE
Canvas freezes for one point project

### ISSUE THREAD

**Issue by edips** [lutraconsulting/input#553]
Hello. I tested Input with a point layer project. It includes one point. CRS of the point layer is Pseudo Mercator. When opening first, it can zoom-in, zoom-out. But when I press "Zoom to Project" the map canvas freezes. "Zoom to Layer" works when I add second point. 
Another problem is when I use empty qgs project template, it doesn't zoom-in, zoom-out or extent. Is there any way to fix it? Thanks.

Tested on Android 8.0 with input 0.4.4
[point1.zip](https://github.com/lutraconsulting/input/files/3926363/point1.zip)

**Comment by saberraz** [comment]
Confirmed. I cannot see my position either.

**Comment by edips** [comment]
There can be a feature counter function when loading a project. If there is a one point feature, the current extent can be set to x+10, x-10, y+10, y-10. This function can be used onClicked event for Zoom to Layer action.

**Comment by edips** [comment]
I debugged extent of the point, it was "Empty". After adding this code to [Loader's](https://paste.ubuntu.com/p/jGRMbzSWp4/) `zoomToProject` function "Zoom to Project" and GPS mode work:
```
if(extent.isEmpty()){
     extent.combineExtentWith( QgsRectangle(10.0,10.0,20.0,20.0) );
  }
```

**Comment by saberraz** [comment]
Do you want to do a Pull Request with your changes?

**Comment by edips** [comment]
I have old input repository, I will update it and I will do a Pull Request.

**Comment by wonder-sk** [comment]
Fixed in #560
